independent-projects/bootstrap/maven-resolver/src/main/java/io/quarkus/bootstrap/resolver/maven/workspace/WorkspaceModulePom.java

```diff
@@ -24,6 +24,10 @@ public WorkspaceModulePom(Path pom, Model model, Model effectiveModel) {
         this.effectiveModel = effectiveModel;
     }
 
+    Path getPom() {
+        return pom;
+    }
+
```

Comment:
It seems like we don't need this method now

Reply:
Right, I dropped it.